thetis/utility.py

```diff
@@ -1271,54 +1271,85 @@ def get_cell_widths_2d(mesh2d):
                   widths[0] = fmax(widths[0], fabs(coords[2*i] - coords[(2*i+2)%6]));
                   widths[1] = fmax(widths[1], fabs(coords[2*i+1] - coords[(2*i+3)%6]));
                 }""", dx, {'coords': (mesh2d.coordinates, READ), 'widths': (cell_widths, RW)})
-    assert cell_widths.vector().gather().min() >= 0.0  # TODO: TEMPORARY
     return cell_widths
 
 
-def get_sipg_ratio(nu):
+def anisotropic_cell_size(mesh):
     """
-    Compute the ratio between the maximum of `nu` and the minimum of `nu` in each element. If `nu`
-    is P0 or a `Constant` then the resulting ratio is unity in each element. If `nu` varies linearly
-    in each element then the ratios are outputted as a P0 field.
+    Measure of cell size for anisotropic meshes, as described in [Micheletti, Perotto & Picasso 2003].
```

Comment:
A full reference? (and keep 1st docstring line short, followed with blank line).